coral/src/app/components/documentation/documentation-editor.module.css

```diff
@@ -0,0 +1,45 @@
+.markdownEditor {
+  position: relative;
+  /*Setting a few shared variables to create styles*/
+  --markdown-editor-line-height: 24px;
+}
+
+.markdownSyntaxHighlight {
+  line-height: var(--markdown-editor-line-height);
+  /*padding-bottom is needed for the short state when user does line break
+    without that they started typing. This would not create a line break, so
+    the cursor would over over the border. This way, making a new line, there
+    will be the visual pedant to a new line added ;*/
+  padding-bottom: calc(var(--markdown-editor-line-height) / 2 + 6px);
+}
+.markdownSyntaxHighlight:focus-within {
+  outline: 2px solid var(--interactive-elements-focus) !important;
+  outline-offset: 0 !important;
+}
+
+:root textarea.markdownTextarea:focus-visible {
+  /*overrides the focus styles to enable sizing we need*/
+  outline: 0 solid transparent !important;
+  outline-offset: 0 !important;
+}
+
+textarea.markdownTextarea {
+  position: absolute;
+  inset: 0;
+  width: 100%;
+  height: 100%;
+  /* 9px exact is needed to overlay text for textarea
+       and syntax highlight correctly. */
+  padding: 9px;
+  color: transparent;
+  border: 2px solid red;
```

Comment:
Is there a reason for the red? It's pretty bright haha
<img width="1209" alt="Screenshot 2023-06-28 at 15 15 25" src="https://github.com/aiven/klaw/assets/20607294/005f15f1-57d5-4a58-9558-93da2ac1c318">

Reply:
It sneaked in during debugging 🙈